web/views/FcRequestStudyBulkView.vue

```diff
@@ -0,0 +1,292 @@
+              :study-request-changes="studyRequestChanges" />
+          </v-col>
+          <v-col cols="6">
+            <FcSummaryStudyRequestBulk
+              class="mx-5"
+              :study-request-bulk="studyRequestBulk"
+              :study-request-users="studyRequestUsers"
+              style="min-height: 320px;" />
+          </v-col>
+          <v-col cols="6">
+            <FcPaneMap
+              class="mx-5"
+              :show-legend="false"
+              :show-location-selection="false"
+              :show-modes="false"
+              :show-search="false" />
+          </v-col>
+        </v-row>
+
+        <div class="mb-6 mx-5">
+          <div class="align-center d-flex px-4 py-2">
+            <v-simple-checkbox
+              v-model="selectAll"
+              class="mr-6"
+              :indeterminate="selectAll === null"></v-simple-checkbox>
+
+            <FcMenu
+              button-class="mr-2"
+              :items="itemsStatus">
+              <span>Mark all as</span>
+            </FcMenu>
+            <FcMenu
+              button-class="mr-2"
+              :items="itemsAssignedTo">
+              <span>Assign all to</span>
+            </FcMenu>
+          </div>
+
+          <v-divider></v-divider>
+
+          <FcDataTableRequests
+            v-model="selectedItems"
+            :columns="columns"
+            :has-filters="false"
+            :items="items"
+            :loading="false"
+            :loading-items="loadingSaveStudyRequest"
+            :sort-by.sync="sortBy"
+            @assign-to="actionAssignTo"
+            @show-item="actionShowItem" />
+        </div>
+      </div>
+    </section>
+  </section>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+import { AuthScope, StudyRequestAssignee } from '@/lib/Constants';
+import { getStudyRequestBulk } from '@/lib/api/WebApi';
+import CompositeId from '@/lib/io/CompositeId';
+import { getStudyRequestItem } from '@/lib/requests/RequestItems';
+import { bulkStatus } from '@/lib/requests/RequestStudyBulkUtils';
+import FcDataTableRequests from '@/web/components/FcDataTableRequests.vue';
+import FcSummaryStudyRequestBulk from '@/web/components/requests/FcSummaryStudyRequestBulk.vue';
+import FcStatusStudyRequestBulk from '@/web/components/requests/FcStatusStudyRequestBulk.vue';
+import FcPaneMap from '@/web/components/FcPaneMap.vue';
+import FcButton from '@/web/components/inputs/FcButton.vue';
+import FcMenu from '@/web/components/inputs/FcMenu.vue';
+import FcMixinAuthScope from '@/web/mixins/FcMixinAuthScope';
+import FcMixinRouteAsync from '@/web/mixins/FcMixinRouteAsync';
+
+export default {
+  name: 'FcRequestStudyBulkView',
+  mixins: [
+    FcMixinAuthScope,
+    FcMixinRouteAsync,
+  ],
+  components: {
+    FcButton,
+    FcDataTableRequests,
+    FcMenu,
+    FcPaneMap,
+    FcStatusStudyRequestBulk,
+    FcSummaryStudyRequestBulk,
+  },
+  data() {
+    // TODO: DRY with Track Requests
```

Comment:
yes, please do - we can move this to its own module under `@/lib/requests`.

I briefly thought about asking for a new `RequestsConstants` module in there, but realized that's a bit too general, and that we'd have to bloat this PR even more to move `StudyRequestStatus` into such a module.  (Well, either that, or leave it as-is and have the confusion of `RequestsConstants` not actually containing all request-related constants / types!)